src/components/atoms/IconStore/IconStore.tsx

```diff
@@ -131,15 +131,13 @@ class IconStore extends React.PureComponent<Props, State> {
 
 
   getLightThemeImage = (iconName: string) => {
-    // return require(IMG_PATH + iconName + ".svg");
-    return (IMG_PATH);
+    return require(IMG_PATH + iconName + ".svg");
```

Comment:
Later on, we can have better handling for extensions as well. To make it more flexible. Can be added as a task in milestone